### ISSUE TITLE
When I shift there is no capital h

### ISSUE THREAD

**Issue by JuniorSum1** [Sorenon/TitleWorlds#10]
Not mutch after that

**Comment by Fishvap** [comment]
What does this mean?

**Comment by JuniorSum1** [comment]
JUst go see for urself when you use the shift button instead of caps there is no h on the virtual keyboard just two gs

**Comment by JuniorSum1** [comment]
Oh and caps do work just not shift

**Comment by IzzyDotExe** [comment]
This is not an issue with titleworlds.

**Comment by JuniorSum1** [comment]
do it urself

**Comment by JuniorSum1** [comment]
i can do it ill show u

**Comment by JuniorSum1** [comment]
Here

**Comment by JuniorSum1** [comment]
Oh wait

**Comment by JuniorSum1** [comment]
I can't put videos so I will put screenshots
![Screenshot (16)](https://user-images.githubusercontent.com/110058148/181106247-844f258d-9bc4-4ea5-bd0d-f5c321f13778.png)
![Screenshot (17)](https://user-images.githubusercontent.com/110058148/181106248-0e8c9e5e-8559-48fb-8eaa-b62eff54a4c2.png)
![Screenshot (18)](https://user-images.githubusercontent.com/110058148/181106249-8467c26b-3059-49b4-b983-51695128cd5f.png)

**Comment by JuniorSum1** [comment]
> This is not an issue with titleworlds.

It is an issue with me and I like to use shift IDK why people sometimes use caps

**Comment by Fishvap** [comment]
No, he's saying this issue doesn't relate to Titleworlds. You're on the wrong Github repository.

**Comment by JuniorSum1** [comment]
then where do i post it i put it in the modrinth issues link

**Comment by Fishvap** [comment]
Titleworlds does not add an in-game keyboard.

**Comment by JuniorSum1** [comment]
thats not my point just look at the screen shots

**Comment by IzzyDotExe** [comment]
This is an issue with MCXR not titleworlds.

**Comment by JuniorSum1** [comment]
U sure cause titleworlds (from what I've heard) is the mod that makes the virtual keyboard for VR users

**Comment by IzzyDotExe** [comment]
Dunno where you heard that. The VR keyboard appears only when you have the VR mod, I would know I literally made the keyboard. 

Titleworlds is a separate mod from MCXR. Please report this as an issue in MCXR.

**Comment by JuniorSum1** [comment]
So then what does title worlds do? I only went on steam VR and went to "Show Desktop" or something like it and then I pressed all the buttons. So do I need TitleWorlds in the first place?
